### DISCUSSION TITLE
this package commercial use

### DISCUSSION THREAD

**Discussion by madfortech** [misterdebug/crud-generator-laravel#27] [Q&A] [answered]
I am using this package for my project and later project sell marketplace ex: themeforest.net.
can I have rights to sell and earn money.

**Comment by misterdebug** [comment]
Do you have a question?

**Reply by madfortech** [reply]
I am asking you.

**Reply by misterdebug** [reply, marked as answer]
You can do what you want as long as you respect MIT license
